exporters/pelorus/config/DEVELOPING.md

```diff
@@ -0,0 +1,51 @@
+# How does this work?
```

Comment:
This seems like an odd way to begin the documentation. Perhaps start with some initial context vs. "this"

Reply:
I figured that folks wouldn't _start_ with DEVELOPING.md, but additional context never hurts.